### PR TITLE
Feat(media): add internal media

### PR DESCRIPTION
**Pull request by ayocodingit**
## Overview

HTTP Private
`/v1/media/:idSetting/`: URL to get all media By Id Setting
`/v1/media/:idSetting/`: URL to post media to Store in the database and upload s3
`/v1/media/:idSetting/:idMedia`: URL to get detail by id Media

image

Todo
- upload cloud storage (S3)

## Evidence
- title: feat(media): add new module media
- project: J-Site
- participants: @ayocodingit @rachadiannovansyah @rindibudiaramdhan

### CODE REVIEW

**Review by rachadiannovansyah — APPROVED** [review]

File: src/database/mongo/schemas/page.schema.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ const schema = new Schema(
         },
         is_active: {
             type: Boolean,
-            index: true,
+            index: false,
```
Comment (rindibudiaramdhan) [inline src/database/mongo/schemas/page.schema.ts:28]: why this is changed, kang @ayocodingit ?

Comment (ayocodingit) [reply src/database/mongo/schemas/page.schema.ts]: thanks, this should not change 🙏

**Review by rindibudiaramdhan — APPROVED** [review]
is log monitoring for these endpoints has been built?

**Comment by ayocodingit** [comment]
> is log monitoring for these endpoints has been built?

Yes i have added log all endpoint

**Comment by ayocodingit** [comment]
> is log monitoring for these endpoints has been built?

Yes i have added log all endpoint

**Pull request merged**